### PR TITLE
US5154 - Header

### PR DESCRIPTION
**Pull request by seancdavis**
Add styles to support a header, including a mobile menu, that matches the current crossroads.net header.

### CODE REVIEW

File: assets/stylesheets/layouts/_header.scss [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  }
+
+  // Dropdown menu container
+  .dropdown-menu {
+    margin-left: 0.5rem;
+    margin-right: 0.5rem;
+    padding: 1.25rem 0;
+    right: 0;
+    top: 2.5rem;
+    box-shadow: 0 6px 9pt $cr-gray-lighter;
```
Comment (zarahzachz) [inline assets/stylesheets/layouts/_header.scss]: Did you mean to have `9pt` here?

File: assets/stylesheets/layouts/_header.scss [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  // Dropdown menu lists
+  .dropdown-menu ul {
+    @extend .list-unstyled;
+    li a {
+      display: block;
+      padding: .1875rem 1.25rem;
+      color: $cr-gray-dark;
+      &:hover {
+        text-decoration: none;
+        background-color: #f5f5f5;
```
Comment (zarahzachz) [inline assets/stylesheets/layouts/_header.scss]: You may want to use a Sass color function on one of our defined gray colors here. Also, I believe dropdown links can be defined using the Bootstrap variables. `$dropdown-link-color` and `$dropdown-link-hover-color` are available and may give you the effect you're looking for.

File: assets/stylesheets/layouts/_header.scss [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+      font-weight: 700;
+    }
+  }
+
+  // Adds a background color to the featured section.
+  .featured-row:before {
+    display: block;
+    @include position(absolute, 0 0 0 0);
+    width: 25%;
+    background: #eeeeee;
```
Comment (zarahzachz) [inline assets/stylesheets/layouts/_header.scss]: Hard-coded hex should be a variable if possible.

**Comment by zarahzachz** [comment]
Another comment for the look of the header - it looks great, but the padding around the Crossroads logo is throwing off the symmetry of the container. I think if you played around with the logo's left-padding it'll look centered.

File: assets/stylesheets/layouts/_header.scss (hunk 1 above)
Comment (seancdavis) [inline assets/stylesheets/layouts/_header.scss]: Shoot. Nope! Copied that over from the crds.net.

**Pull request merged**